xGame/Java/xgame/src/main/java/com/xgame/service/ChessMatchService.java

```diff
@@ -41,6 +48,42 @@ public MatchViewModel createMatch(int whiteId, int blackId) {
 		
 		return null;
 	}
+	
+	@Override
+	public List<MatchInviteViewModel> getInvites(int userId) {
+		var matches = matchRepo.findByBlackPlayerIdAndMatchStatus(userId, MatchStatus.PENDING);
+		
+		return matches
+				.stream()
+				.map(m -> new MatchInviteViewModel(m.getId(), m.getWhitePlayer().getNickname(), m.getCreationTimestamp()))
+				.collect(Collectors.toList());
+	}
+	
+	@Override
+	public boolean acceptInvite(int matchId) {
+		var match = matchRepo.findById(matchId);
+		
+		if(match.isPresent()) {
+			//TODO: initialize chess board
+			
+			var m = match.get();
+			m.setTurnCount(1);
+			m.setMatchStatus(MatchStatus.INPROGRESS);
+			m.setStartTimestamp(new Timestamp(System.currentTimeMillis()));
```

Comment:
I think these lines should be a method on the match object, something like "m.startMatch()"

Reply:
All of the entities are strictly data access objects and will only ever have member variables, constructors, getters, and setters.